### PR TITLE
Add kubelogin plugin for connecting kubectl.

### PR DESCRIPTION
**Pull request by tibeer**
**What this PR does / why we need it**:

`gardenlogin` was not enough for me. I also add to install `kubelogin` to be able to connect to my clusters.

**Which issue(s) this PR fixes**:
None

**Special notes for your reviewer**:

**Release note**:

```bugfix user
NONE
```

### CODE REVIEW

**Comment by gardener-robot-ci-1** [comment]
Thank you @tibeer for your contribution. Before I can start building your PR, a member of the organization must set the required label(s) {'reviewed/ok-to-test'}. Once started, you can check the build status in the PR checks section below.

File: docs/usage/connect-kubectl.md [hunk 1]
```diff
@@ -30,6 +30,7 @@ On this page:
 3. If `gardenlogin` is not installed or configured, click on the *show gardenlogin info* action to follow the installation and configuration hints.
 
    <img width="700" src="../images/03-gardenlogin-info.png">
+4. You might also need to install [kubelogin](https://github.com/int128/kubelogin)
```
Comment (petersutter) [inline docs/usage/connect-kubectl.md:33]:
```suggestion
4. You might also need to install [kubelogin](https://github.com/int128/kubelogin#setup)
```

Comment (tibeer) [reply docs/usage/connect-kubectl.md]: Thanks!

**Review by petersutter — APPROVED** [review]
/lgtm

**Review by grolu — APPROVED** [review]
/lgtm

**Review by holgerkoser — APPROVED** [review]
/lgtm

**Comment by petersutter** [comment]
> [CLA assistant check](https://cla-assistant.io/gardener/dashboard?pullRequest=1421) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/gardener/dashboard?pullRequest=1421) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/gardener/dashboard?pullRequest=1421) it.</sub>

@tibeer

**Pull request merged**